source/publishing_data/01_creating_a_dataset/sourcing_data.rst

```diff
@@ -128,3 +125,7 @@ This method consists in pasting data in a CSV format. This method is most useful
 2. Click on the **Add a source** button.
 3. In the **Paste data** area, write or paste data in a CSV format.
 4. Click on the **Send data** button.
+
+
+Once a new dataset is created, the platform automatically displays an edition interface for that dataset. It is accessible for any user with the right permissions.
```

Comment:
Maybe specify the right permission is either "create dataset" (on the domain) or "edit dataset" (on the domain or this dataset)?